src/cryptoadvance/specter/user.py

```diff
@@ -70,13 +70,19 @@ def __init__(
         password_hash,
         config,
         specter,
+        jwt_token_id,
+        jwt_token,
```

Comment:
Just a reminder that this can be deleted. It can even be done now, no?

Reply:
> Just a reminder that this can be deleted. It can even be done now, no?

By deleting mean do you mean the `delete` route?

Reply:
No, i mean that you delete line 73 and 74 and all the occurrences of those. You now have a `jwt_tokens` and you no longer need them, right?!